contracts/ExoticPositionalMarkets/OraclePausable.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.8.0;
+
+// Inheritance
+import "../utils/proxy/solidity-0.8.0/ProxyOwned.sol";
+import "../interfaces/IExoticPositionalMarketManager.sol";
+
+// Clone of syntetix contract without constructor
+
+contract OraclePausable is ProxyOwned {
+    uint public lastPauseTime;
+    bool public paused;
+
+    /**
+     * @notice Change the paused state of the contract
+     * @dev Only the contract owner may call this.
+     */
+    function setPaused(bool _paused) external pauserOnly {
+        // Ensure we're actually changing the state before we do anything
+        if (_paused == paused) {
+            return;
+        }
+        if (paused) {
+            require(msg.sender == IExoticPositionalMarketManager(owner).owner(), "Only Protocol DAO can unpause");
+        }
+        // Set our paused state.
+        paused = _paused;
+
+        // If applicable, set the last pause time.
+        if (paused) {
+            lastPauseTime = block.timestamp;
+        }
+
+        // Let everyone know that our pause state has changed.
+        emit PauseChanged(paused);
+    }
+
+    event PauseChanged(bool isPaused);
+
+    modifier notPaused {
```

Comment:
maybe its best to use a different modifier in case we need to combine it with normal pause

Reply:
it is like normal, but only pDAO can unpause it